packages/rke2-coredns/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,14 +1,5 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -2,7 +2,7 @@
-   artifacthub.io/changes: |
```

Comment:
Was removing this bit intentional?

Reply:
Reviewing patches to patches... we're not patching it out, it was removed from the upstream Chart.yaml. Our only change remains the appVersion.